src/components/ResidentDocumentsTable.tsx

```diff
@@ -209,9 +211,9 @@ export const ResidentDocumentsTable: FunctionComponent<Props> = ({
                               <li
                                 className={styles.item}
                                 data-testid={`${documentTab.id}-evidence-tile`}
+                                key={index}
                               >
                                 <EvidenceTile
```

Comment:
Does this one need a key?

Reply:
Yeah it's being mapped so needs a key otherwise React will complain.